packaging/technical-addon/packaging-scripts/cicd-tests/envvars/envvartest.sh

```diff
@@ -0,0 +1,67 @@
+
+MAX_ATTEMPTS=6
+DELAY=10
+ATTEMPT=1
+while [ $ATTEMPT -le $MAX_ATTEMPTS ]; do
+    if docker exec -u root envvars-ta-test-envvars-1 grep -qi "Everything is ready. Begin running and processing data." /opt/splunk/var/log/splunk/otel.log; then
+        break
+    else
+        if [ $ATTEMPT -eq $MAX_ATTEMPTS ]; then
+            echo "Failed to see success message in otel.log after $MAX_ATTEMPTS attempts."
+            docker exec -u root envvars-ta-test-envvars-1 cat /opt/splunk/var/log/splunk/otel.log
+            exit 1
+        fi
+        echo "success message not found in otel.log Retrying in $DELAY seconds"
+        ATTEMPT=$((ATTEMPT + 1))
+        sleep $DELAY
+    fi
+done
+
+echo "checking env vars"
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_given: Str(helloworld)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_GOMEMLIMIT: Str(512MiB)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_ACCESS_TOKEN_FILE: Str(/opt/splunk/etc/apps/Splunk_TA_otel/local/access_token)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_API_URL: Str(https://api.us0.signalfx.com)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_BUNDLE_DIR: Str(/opt/splunk/etc/apps/Splunk_TA_otel/linux_x86_64/bin/agent-bundle)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_COLLECTD_DIR: Str(/opt/splunk/etc/apps/Splunk_TA_otel/linux_x86_64/bin/agent-bundle/run/collectd)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_CONFIG: Str(/opt/splunk/etc/apps/Splunk_TA_otel/configs/passthrough_env_vars.yaml)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_DEBUG_CONFIG_SERVER: Str(test_notused)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_GATEWAY_URL: Str(test_notused)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_HEC_URL: Str(test_notused)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_INGEST_URL: Str(https://ingest.us0.signalfx.com)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_LISTEN_INTERFACE: Str(localhost)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_OTEL_LOG_FILE_NAME: Str(/opt/splunk/var/log/splunk/otel.log)' /opt/splunk/var/log/splunk/otel.log
+docker exec -u root envvars-ta-test-envvars-1 grep -qi 'envvartest_SPLUNK_REALM: Str(us0)' /opt/splunk/var/log/splunk/otel.log
```

Comment:
could we replace this with a for loop instead for better readability?
```suggestion
for var in \
    'envvartest_given: Str(helloworld)' \
    'envvartest_GOMEMLIMIT: Str(512MiB)' \
    ...
    'envvartest_SPLUNK_REALM: Str(us0)'; do
    docker exec -u root envvars-ta-test-envvars-1 grep -qi "$var" /opt/splunk/var/log/splunk/otel.log
done
```